std/root.ln

```diff
@@ -660,212 +660,217 @@ export fn toHashMap(kva: Array<KeyVal<Hashable, any>>) {
-  const iters = length(n.tree.parents)
+  const iters = length(n.tree.parents);
   seqeach(newseq(iters), fn (i: int64) {
-    const parentId = getOr(n.tree.parents[i], -1)
-    const parentDelta = rmdIds.filter(fn (rmId: int64): bool = parentId > rmId).length()
+    const parentId = getOr(n.tree.parents[i], -1);
+    const parentDelta = rmdIds.filter(fn (rmId: int64): bool = parentId > rmId).length();
     if parentDelta > 0 {
-      set(n.tree.parents, i, parentId - parentDelta)
+      set(n.tree.parents, i, parentId - parentDelta);
     }
-    const children = getOr(n.tree.children[i], new Array<int64> [])
+    const children = getOr(n.tree.children[i], new Array<int64> []);
     const newChildren = children.map(fn (cId: int64): int64 {
-      const delta = rmdIds.filter(fn (rmdId: int64): bool = cId > rmdId).length()
+      const delta = rmdIds.filter(fn (rmdId: int64): bool = cId > rmdId).length();
       if delta > 0 {
-        return cId - delta
+        return cId - delta;
       }
-      return cId
-    })
-    set(n.tree.children, i, newChildren)
-  })
-  return n.tree
+      return cId;
+    });
+    set(n.tree.children, i, newChildren);
+  });
+  return n.tree;
 }
 
-export fn getChildren(t: Tree<any>): Array<Node<any>> = t.getRootNode().getChildren()
+export fn getChildren(t: Tree<any>): Array<Node<any>> = t.getRootNode().getChildren();
 
 // returns the new child node added
 export fn addChild(n: Node<any>, val: any): Node<any> {
-  const childId = length(n.tree.vals)
-  push(n.tree.vals, val)
-  push(n.tree.parents, n.id)
-  push(n.tree.children, new Array<int64> [ ])
-  push(getOr(n.tree.children[n.id], new Array<int64> []), childId)
+  const childId = length(n.tree.vals);
+  push(n.tree.vals, val);
+  push(n.tree.parents, n.id);
+  push(n.tree.children, new Array<int64> [ ]);
+  push(getOr(n.tree.children[n.id], new Array<int64> []), childId);
   return new Node<any> {
-    id = childId
-    tree = n.tree
-  }
+    id: childId,
+    tree: n.tree,
+  };
 }
 
-export fn addChild(t: Tree<any>, val: any): Node<any> = t.getRootNode().addChild(val)
+export fn addChild(t: Tree<any>, val: any): Node<any> = t.getRootNode().addChild(val);
 
-export fn addChild(t: Tree<any>, val: Node<any>): Node<any> = t.getRootNode().addChild(val)
+export fn addChild(t: Tree<any>, val: Node<any>): Node<any> = t.getRootNode().addChild(val);
 
-export fn addChild(n: Node<any>, val: Tree<any>): Node<any> = n.addChild(val.getRootNode())
+export fn addChild(n: Node<any>, val: Tree<any>): Node<any> = n.addChild(val.getRootNode());
 
-export fn getOr(n: Node<any>, default: any): any = getOr(n.tree.vals[n.id], default)
+export fn getOr(n: Node<any>, default: any): any = getOr(n.tree.vals[n.id], default);
 
-export fn toNodeArray(t: Tree<any>): Array<Node<any>> = map(t.vals, fn (val: any, i: int64): Node<any> = t.getNodeById(i))
+export fn toNodeArray(t: Tree<any>): Array<Node<any>> = map(
+  t.vals,
+  fn (val: any, i: int64): Node<any> = t.getNodeById(i)
+);
```

Comment:
Take a look at this. It seems trivial, but this kind of thing wasn't parsing correctly before!